src/common/modal/modal.module.scss

```diff
@@ -0,0 +1,38 @@
+@import 'fonts';
+@import 'spacings';
+@import 'colours';
+@import 'z-index';
+
+.overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: transparent;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  z-index: z-index(modal);
+}
+
+.modal {
+  background-color: $white;
+  position: relative;
+  box-shadow: 0 0 5px $black;
+  &:focus {
+    outline: none;
+  }
+}
+
+.modalContent {
+  background-color: $white;
+  padding: units(3);
+  display: flex;
+  flex-direction: column;
+
+  h4 {
```

Comment:
I prefer exposing the className of the Section's title instead of targeting `h4` directly because it won't reach it if `h4` was replaced for with something else.

Or you might consider using `<Text as='h4' color='brand' ... />` instead of `<Section ...`

Reply:
Good point, I'll change this one in my branch

Reply:
Haven't modified the modal's styles in this branch.